packages/federation/test/defer.test.ts

```diff
@@ -1,49 +1,19 @@
+import { exec } from 'child_process';
```

Comment:
I think this is extra, probably that's why `tsc` fails.

Reply:
thanks!

Reply:
done in 1ce4b1ef